### ISSUE TITLE
Support of nested SSPs

### ISSUE THREAD

**Issue by robha67** [OpenModelica/OMSimulator#1146]
### TODO

- [ ] @robha67 make a manual example of a nested SSP

**Comment by robha67** [comment]
This is a feature request that likely will be needed by the vehicle systems demonstrator in the OpenScsling project @arun3688

**Comment by arun3688** [comment]
@robha67 ok sure we need a meeting to discuss this in more detail

**Comment by robha67** [comment]
@arun3688 i agree, i am away next week but we can try to set something up in two weeks?

**Comment by robha67** [comment]
@lochel I am not able to change the label

**Comment by lenaRB** [comment]
1st step - @robha67 - make a manual example of a nested SSP

**Comment by lenaRB** [comment]
progress made on how to specify nested SSP in the SSP standard - we know how to find that information in the specification -[link to documentation](https://ssp-standard.org/docs/2.0/#_system)
